relengapi/blueprints/tooltool/grooming.py

```diff
@@ -131,13 +134,16 @@ def check_pending_upload(session, pu):
     # (after which the user can't make any more changes, but the upload
     # may yet be incomplete) and 1 day afterward (ample time for the upload
     # to complete)
+    sha512 = pu.file.sha512
+    size = pu.file.size
+
     if time.now() < pu.expires:
         # URL is not expired yet
         return
     elif time.now() > pu.expires + timedelta(days=1):
         # Upload will probably never complete
         log.info(
-            "Deleting abandoned pending upload for {}".format(pu.file.sha512))
+            "Deleting abandoned pending upload for {}".format(sha512))
         session.delete(pu)
```

Comment:
ok, now with my thinking cap on correctly, the return on the next line makes *this* comment valid.